src/resample.c

```diff
@@ -51,7 +52,7 @@ main (int argc, char *argv [])
 		exit (0) ;
 		} ;
 
-	if (argc != 5 && argc != 7 && argc != 8)
+	if (argc != 5 && argc != 7 && argc != 8 && argc != 9 && argc != 10 && argc != 11)
```

Comment:
Maybe we need `argc < 5 || argc == 6 || argc > 11`.

Reply:
done